### PR TITLE
exportGraphics: add backend test to verify uv-parameter output

### PR DESCRIPTION
**Pull request by dassaf4**
Improvements due to an `exportGraphics` investigation.

Mainly, this PR adds a backend test to verify known uv-parameter output by `exportGraphics` for specific textured sample geometry with stored uv-parameters.

Also:
* `GrowableXYArray`: add `pushFrom`, expand input type for `create`, test
* `GrowableXYZArray`: add `createArrayOfGrowableXYZArray` and deprecate the original version in GrowableXYArray where it was misplaced
* add VSCode Run & Debug config for export-gltf
* narrow the return type of `Loop.cloneStroked` and `Path.cloneStroked`
* documentation

Note:
* The new flag `ExportGraphicsOptions.exportRawParameters` proposed in a linked pair of closed iTwin/imodel PRs has been abandoned as unnecessary. Each texture specifies how the raw uv-parameters of a geometry are to be transformed.

### CODE REVIEW

**Review by saeeedtorabi — APPROVED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -616,6 +616,21 @@
...
+      "name": "export-gltf",
+      "presentation": {
+        "group": "6_Misc"
+      },
+      "cwd": "${workspaceFolder}/test-apps/export-gltf",
+      "type": "node",
+      "request": "launch",
+      "program": "${workspaceFolder}/test-apps/export-gltf/lib/ExportGltf.js",
+      "args": [   // customize your input/output below
+        "--input=d:\\triangleWithUVParams.bim",
```
Comment (calebmshafer) [inline .vscode/launch.json:629]: is this iModel and gltf somewhere in the repo?

Comment (calebmshafer) [reply .vscode/launch.json:629]: I'd suggest removing that from the args committed. Otherwise PR looks good.

Comment (dassaf4) [reply .vscode/launch.json:629]: Thanks. Neither are in the repo, but I think it needs a placeholder to show how to customize. What about "d:\\foo.bim" and "d:\\foo.gltf"?

Comment (calebmshafer) [reply .vscode/launch.json]: using "foo" or "replaceme" as the name is good with me.

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**